bin/code_climate_reek

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env ruby
+#
+# Wrapper for the CodeClimate integration.
```

Comment:
Since this file exists, would it make sense to not load the codeclimate-specific code when just running `reek`? It means we can also drop the codeclimate-engine-rb dependency I think. Of course, we would then have to make sure it is loaded inside the Docker container some other way.